phpunit.xml

```diff
@@ -13,15 +13,13 @@
             <directory>tests/</directory>
         </testsuite>
     </testsuites>
-
-    <logging>
-        <log type="coverage-html" target="tests/coverage/"/>
-    </logging>
-
     <filter>
         <whitelist>
             <directory suffix=".php">src/</directory>
         </whitelist>
+        <blacklist>
```

Comment:
this is not useful as we have a whitelist filter (and "vendor" is not in "src"). you can revert